pkg/events/recorder/recorder.go

```diff
@@ -67,7 +68,12 @@ func (sr *SimpleRecorder) makeEvent(ref *v1.ObjectReference, labels map[string]s
 	t := metav1.NewTime(now)
 	namespace := ref.Namespace
 	if namespace == "" {
-		namespace = metav1.NamespaceDefault
+		if key := os.Getenv("NAMESPACE"); key != "" {
+			namespace = os.Getenv("NAMESPACE")
```

Comment:
no need to use os.Getenv to retrieve NAMESPACE again,  use 
```
namespace = key  
```
is enough.

Reply:
Yes.